pyramid/config/views.py

```diff
@@ -122,7 +122,7 @@ def add(self, view, order, accept=None, phash=None):
                     self.views[i] = (order, view, phash)
                     return
 
-        if accept is None or '*' in accept:
```

Comment:
We don't even try to do the right thing here, if accept were `text/*` then they wouldn't even get counted...